operator/option/config.go

```diff
@@ -322,6 +330,7 @@ func (c *OperatorConfig) Populate() {
 
 	c.AWSReleaseExcessIPs = viper.GetBool(AWSReleaseExcessIPs)
 	c.UpdateEC2AdapterLimitViaAPI = viper.GetBool(UpdateEC2AdapterLimitViaAPI)
+	c.UpdateEC2AdapterLimitViaAPIv2 = viper.GetBool(UpdateEC2AdapterLimitViaAPIv2)
```

Comment:
No need to introduce a new option `OperatorConfig.UpdateEC2AdapterLimitViaAPIv2` for this. Just assign the existing `OperatorConfig.UpdateEC2AdapterLimitViaAPI` and make the new option flag override the deprecated one in the unlikely case that both are specified.

Reply:
Ok makes sense, will do.

Reply:
Done.